desktop/src/Desktop/Setup.hs

```diff
@@ -163,15 +163,15 @@ walletSetupRecoverHeader currentScreen = setupDiv "workflow-header" $ do
 
     isActive sid = sid `elem` (progress currentScreen)
     progress WalletScreen_Password =
```

Comment:
This would better be represented with partial orders, but `lattices` is giving me cabal hell right now

Reply:
Actually, nevermind, even defining `leq` seems too boilerplate-ish.